### ISSUE TITLE
Add tab completion to jabba

### ISSUE THREAD

**Issue by leonard84** [shyiko/jabba#589]
I've created a small PowerShell module for jabba https://gist.github.com/leonard84/3828a2584e6a3cbc30a90c44c86b191e

It uses dedicated functions, e.g. `Jabba-Use` as this makes the auto completion easier.

![JabbaExt-Demo](https://user-images.githubusercontent.com/1151511/72344426-7c517780-36d1-11ea-995f-2c68998c7366.gif)

Here is the documentation on how to install a PowerShell module https://docs.microsoft.com/en-us/powershell/scripting/developer/module/installing-a-powershell-module?view=powershell-7

It must be put here `$Home\Documents\WindowsPowerShell\Modules` (%UserProfile%\Documents\WindowsPowerShell\Modules)
> A well-formed module is a module that is stored in a directory that has the same name as the base name of at least one file in the module directory. If a module is not well-formed, Windows PowerShell does not recognize it as a module.
>
>The "base name" of a file is the name without the file name extension. In a well-formed module, the name of the directory that contains the module files must match the base name of at least one file in the module.

In short it must be put in `$Home\Documents\WindowsPowerShell\Modules\JabbaExt\JabbaExt.psm1`

**Comment by leonard84** [comment]
@willcohen since a release is in sight, WDYT about adding this?

**Comment by willcohen** [comment]
Hi there -- I'm not personally a Windows user and am unfamiliar with PowerShell, so I'm not sure where to go from here. A PR that incorporates this would be very useful!

**Comment by leonard84** [comment]
I'm unsure how to best add this to the installation, basically the [file](https://gist.github.com/leonard84/3828a2584e6a3cbc30a90c44c86b191e) must be put in `$Home\Documents\WindowsPowerShell\Modules\JabbaExt\JabbaExt.psm1`.

Should I just inline it into the `install.ps1` script or would you add the module to the release so that it can be downloaded like the exe?
